terraform/aws/scripts/install.sh.tpl

```diff
@@ -6,7 +6,13 @@ sudo apt-get update -y
 sudo apt-get install -y curl unzip
 
 # Download Vault into some temporary directory
-curl -L "${download_url}" > /tmp/vault.zip
+vault_download_url=https://releases.hashicorp.com/vault/${download_version}/vault_${download_version}_linux_amd64.zip
```

Comment:
It's probably worth wrapping this URL in double quotes, since there are variables in it.  See https://github.com/koalaman/shellcheck/wiki/SC2086 for more details.